build.sbt

```diff
@@ -445,10 +445,8 @@ lazy val unit = project
     testSettings,
     libraryDependencies ++= List(
       "io.get-coursier" %% "coursier" % V.coursier, // for jars
-      "org.scalameta" %% "testkit" % V.scalameta,
```

Comment:
I also took the liberty of removing this: it's not used anymore since we introduced MUnit